Doc/using/configure.rst

```diff
@@ -747,6 +747,17 @@ Compiler flags
    extensions.  Use it when a compiler flag should *not* be part of the
    distutils :envvar:`CFLAGS` once Python is installed (:issue:`21121`).
 
+   In particular, :envvar:`CFLAGS` should not contain:
+
+   - the compiler flag `-I` (for setting the search path for include files);
```

Comment:
I believe that instead of `-`, the convention is `*` instead

Reply:
Thanks, done